viadot/sources/databricks.py

```diff
@@ -428,3 +428,61 @@ def discover_schema(self, schema: str, table: str) -> dict:
         schema = dict(zip(col_names, data_types))
 
         return schema
+
+    def get_table_ver(self, schema: str, table: str) -> str:
+        """
+        Get the current table's version number.
+
+        Args:
+            schema (str): Name of the schema.
+            table (str): Name of the table to rollback.
+
+        Returns:
+            int: The table's version number
+        ```
+        """
+        fqn = f"{schema}.{table}"
+
+        # Get the info regarding the current version of the delta table
+        history = self.run(f"DESCRIBE HISTORY {fqn}", "pandas")
+
+        # Extract the current version number
+        ver_num = history["version"].iat[0]
+        return ver_num
+
+    def rollback(self, schema: str, table: str, ver_num: int):
+        """
+        Rollback the table to a previous version.
+
+        Args:
+            schema (str): Name of the schema.
+            table (str): Name of the table to rollback.
+            ver_num (int): Number of the table's version to rollback to.
+
+        Example:
+        ```python
+        from viadot.sources import Databricks
+        databricks = Databricks()
+
+        schema = "test"
+        table = "table_1"
+
+        ver_num = databricks.get_table_ver(schema = schema, table = table)
+
+        # Perform changes on the table, in this example we are appending to the table
+        list = [{"id":"1", "name":"Joe"}]
+        df = pd.DataFrame(list)
+        databricks.insert_into(schema = schema, table = table, df = df, if_exists="append")
+
+        databricks.rollback(schema, table, ver_num)
+        ```
+        """
+        fqn = f"{schema}.{table}"
```

Comment:
This will break if schema is None. You should check for that and use `Databricks.DEFAULT_SCHEMA` in such case.